plugins/arElasticSearchPlugin/config/mapping.yml

```diff
@@ -74,6 +74,16 @@ mapping:
       subject_id: { type: integer, include_in_all: false }
       type_id: { type: integer, include_in_all: false }
 
+  accession_event:
+    _attributes:
+      i18n: true
+      i18nExtra: [property, term]
+      nested_only: true
+    dynamic: true
```

Comment:
Why dynamic? I think it's better to have full control over the mapping fields.

PS. I don't know why we have it on the `donors` too but all others are set to strict.

Reply:
Fixed!